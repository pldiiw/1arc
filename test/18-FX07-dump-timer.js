'use strict';

let engine = require('engine').initialize();
const dumpTimer = require('instructions').dumpTimer;

engine.data[2] = 120;
engine.timer = 230;

const engine1 = dumpTimer(engine, 2);

console.log(engine.data[2]);
console.log(engine.timer);
console.log(engine1.data[2]);
console.log(engine1.timer);
