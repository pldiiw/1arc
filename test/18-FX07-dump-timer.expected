120
230
230
230
